pmsv2/virtualaccount/params.go

```diff
@@ -0,0 +1,54 @@
+package virtualaccount
+
+import (
+	"time"
+
+	"github.com/xendit/xendit-go/pmsv2/constant"
+)
+
+type ChannelCode string
+
+const (
+	BCA               ChannelCode = "BCA"
+	BJB               ChannelCode = "BJB"
+	BNI               ChannelCode = "BNI"
+	BRI               ChannelCode = "BRI"
+	MANDIRI           ChannelCode = "MANDIRI"
+	PERMATA           ChannelCode = "PERMATA"
+	BSI               ChannelCode = "BSI"
+	CIMB              ChannelCode = "CIMB"
+	SAHABAT_SAMPOERNA ChannelCode = "SAHABAT_SAMPOERNA"
+	ARTAJASA          ChannelCode = "ARTAJASA"
+	PV                ChannelCode = "PV"
+	VIETCAPITAL       ChannelCode = "VIETCAPITAL"
+	WOORI             ChannelCode = "WOORI"
+)
+
+type ChannelProperties struct {
+	CustomerName         string     `json:"customer_name"`
+	VirtualAccountNumber *string    `json:"virtual_account_number"`
+	ExpiresAt            *time.Time `json:"expires_at"`
+	SuggestedAmount      float64    `json:"suggested_amount,omitempty"`
+}
+
+type CreateMethod struct {
+	Amount            float64               `json:"amount"`
+	MinimumAmount     float64               `json:"min_amount,omitempty"`
+	MaximumAmount     float64               `json:"max_amount,omitempty"`
+	Currency          constant.CurrencyEnum `json:"currency"`
+	ChannelCode       ChannelCode           `json:"channel_code"`
+	ChannelProperties ChannelProperties     `json:"channel_properties"`
+	VirtualAccountID  string                `json:"virtual_account_id"`
```

Comment:
can remove, since this is an internal field

Reply:
removed. Though, this is also available in the CreateMethod of PMSv2. Is that fine?
![image](https://user-images.githubusercontent.com/24637976/207242892-fa593fd8-95b5-490c-8bb3-8f1d7915ab7f.png)

Reply:
hmm that's a bug then, ideally it should only be available in internal GET response.